llarp/config/key_manager.cpp

```diff
@@ -186,7 +186,7 @@ namespace llarp
   bool
   KeyManager::backupKeyFilesByMoving() const
   {
-    std::vector<std::string> files = {m_rcPath, m_idKeyPath, m_encKeyPath, m_transportKeyPath};
+    std::vector<std::string> files = {m_rcPath.string(), m_idKeyPath.string(), m_encKeyPath.string(), m_transportKeyPath.string()};
```

Comment:
Here, too, makes a lot more sense as a vector of `fs::path` rather than string.  These feel like we are doing the path-to-string conversion way too early by forcing to caller to do it rather than making the callee take a proper `path` object.